blog/en/blog/2022/10/19/deploy-apisix-on-scaleway-cloud.md

```diff
@@ -0,0 +1,293 @@
+{
+  "host": "httpbin.org",
+  "uri": "/*",
+  "upstream": {
+    "type": "roundrobin",
+    "nodes": {
+      "httpbin-external.default.svc.cluster.local:80": 1
+    }
+  }
+}'
+```
+
+Then, we can verify access and if the newly created route is functioning correctly by running another final command:
+
+``` shell
+$ curl http://51.159.10.6:80/get -H 'Host: httpbin.org'
+{
+  "headers": {
+    "Accept": "*/*",
+    "Host": "httpbin.org",
+    "User-Agent": "curl/7.68.0",
+    "X-Amzn-Trace-Id": "Root=1-63228c4e-0388e1b61255180620195210",
+    "X-Forwarded-Host": "httpbin.org"
+  }
+}
+```
+
+Here we go, APISIX Admin API is responding to our requests in the Scaleway Cloud and the request is forwarded to the external service.  
+
+### Deploy and run APISIX Ingress Controller
+
+Next, we will try to install and verify [Apache APISIX Ingress Controller](https://apisix.apache.org/docs/ingress-controller/getting-started/) is running in the Scaleway Cloud and we do very similar steps as we did for APISIX.
+
+To install APISIX Ingress with helm:
+
+``` shell
+$ helm repo add apisix https://charts.apiseven.com
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo update
+
+$ kubectl create ns ingress-apisix
+$ helm install apisix apisix/apisix \
+  --set gateway.type=LoadBalancer \
+  --set ingress-controller.enabled=true \
+  --namespace ingress-apisix \
+  --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix
+
+```
+
+Then, check the deployment status:
+
+``` shell
+$ kubectl get pods -n ingress-apisix
+
+NAME                                         READY   STATUS    RESTARTS   AGE
+apisix-5bcf68b548-qrsqb                      1/1     Running   0          1m
+apisix-etcd-0                                1/1     Running   0          1m
+apisix-etcd-1                                1/1     Running   0          1m
+apisix-etcd-2                                1/1     Running   0          1m
+apisix-ingress-controller-75bd4d9b9b-7xfn5   1/1     Running   0          1m
+
+$ kubectl get svc -n ingress-apisix
+
+apisix-admin                ClusterIP      10.36.120.143   <none>          9180/TCP            82m
+apisix-etcd                 ClusterIP      10.47.39.201    <none>          2379/TCP,2380/TCP   82m
+apisix-etcd-headless        ClusterIP      None            <none>          2379/TCP,2380/TCP   82m
+apisix-gateway              LoadBalancer   10.45.241.11    51.159.206.46   80:30197/TCP        82m
+apisix-ingress-controller   ClusterIP      10.47.236.40    <none>          80/TCP              82m
+```
+
+Afterward, we can deploy mock server `htttbin` to the default namespace in order to test the ingress controller:
+
+``` shell
+kubectl run httpbin --image kennethreitz/httpbin --port 80
+kubectl expose pod httpbin --port 80
+```
+
+Now we are ready to test ingress by creating a new route the same as below
+
+``` shell
+kubectl apply -f - <<EOF
+apiVersion: apisix.apache.org/v2beta3
```

Comment:
we have released APISIX Ingress v1.5, we should change this apiVersion to v2
```suggestion
apiVersion: apisix.apache.org/v2
```

Reply:
@tao12345666333 Right!